tokio/src/io/read_buf.rs

```diff
@@ -39,7 +39,7 @@ impl<'a> ReadBuf<'a> {
         }
     }
 
-    /// Creates a new `ReadBuf` from a fully uninitialized buffer.
+    /// Creates a new `ReadBuf` from a buffer that may be fully uninitialized.
```

Comment:
It could be partially but not fully uninitialized too.
```suggestion
    /// Creates a new `ReadBuf` from a buffer that may be uninitialized.
```